analysis/experiment_results.py

```diff
@@ -72,6 +72,19 @@ def __init__(  # pylint: disable=too-many-arguments
     def _get_full_path(self, filename):
         return os.path.join(self._output_directory, filename)
 
+    def linkify_names(self, df):
+        """For any DataFrame which is indexed by fuzzer names, turns the fuzzer
+        names into links to their descriptions on GitHub."""
+        assert df.index.name == 'fuzzer'
+
+        def description_link(commit, fuzzer):
+            return (f'<a href="https://github.com/google/fuzzbench/blob/'
```

Comment:
Should we check if the file exists, if yes create this link, otherwise point to fuzzer root {commit}/fuzzers/{fuzzer}/

Reply:
Ah, yes, we were saying it's best to just link to the directory, as it's easy to see the description and the config file there. I changed it accordingly.

Reply:
Yes that sounds good, since most people care about code too.